ipcalc-geoip.c

```diff
@@ -34,6 +34,10 @@
 # include <GeoIP.h>
 # include <GeoIPCity.h>
 
+#ifndef GEOIP_SILENCE
+#define GEOIP_SILENCE 0	/* fix libgeoip < 1.6.3 */
+#endif
```

Comment:
That breaks geoip >= 1.6.3. GEOIP_SILENCE is an enumerated value.

Reply:
That doesn't break geoip >= 1.6.3 because I use a ifnedef clause. So when GEOIP_SILENCE is supported/defined by geoip/GeoIP.h (>= 1.6.3), we use the right value of the enumeration.

Using the "define GEOIP_SILENCE 0" is like using an "alias" of GEOIP_STANDARD for GEOIP_SILENCE, so it's a solution here.

So I know, it's look like a hack, but it's the only way I find to make the build works fine on Ubuntu Trusty (used by  Travis).

Reply:
`#ifndef` works only if the GEOIP_SILENCE is a definition. In geoip is an enumeration. Thus `#ifndef GEOIP_SILENCE` will always be true. What happens if you add unconditionally
`#define GEOIP_SILENCE 16`?